docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -918,4 +918,202 @@ Amplify Ally:
+    Save:
+        Trigger: If the space is occupied, the creature occupying that space makes a 
+        Type: STR
+        Fail: they are pushed out of the way
+        Succeed: instead, choose a space within 1 space of the creature to appear. 
+
+Summon Creature:
+    Type: Major
+    Category:
+        - Companion
+        - Summoner
+    Mechanic: Summon a creature to assist you. This creature uses the 'Summoned Creature' stat block and is treated as a Companion for the duration. You choose the creature's form. 
+    Description: It's good to have a friend, even when you have to summon them from somewhere else. Who ever said no to a little extra help?
+    XP: 2
+    PP: 2
+    Prereq:
+        Role: Caster
+        Level: 2
+        Skill: Knowledge > 0
+    Duration: 1 hour
+
+Phase Swap:
+    Type: Minor
+    Category:
+        - Companion
+        - Movement
+        - Summoner
+    Mechanic: Use half your speed to swap spaces with a Summoned Creature 
+    Description: It can often be better to change places with a friendly creature. You'd prefer to be there, or they'd prefer to be here.
+    XP: 2
+    PP: 1
+    Prereq:
+        Role: Caster
+        Level: 3
+        Skill: Knowledge > 1
+        Power: Summon Creature
+    Range: 10
+
+Life Link: 
+    Type: Passive
+    Category: 
+        - Companion
+        - Summoner   
+    Mechanic: While you have at least one summoned creature active, when you take damage you can choose to apply that damage to your summoned creature instead.
+    Description: Channeling energy between friends can sometimes be mutually beneficial. Other times, it's simply beneficial.
+    XP: 2
+    Prereq:
+        Role: Caster
+        Level: 3
+        Skill: Knowledge > 1
+        Power: Summon Creature
+
+Create Potion:
+    Type: Major
+    Category:
+        - Recovery
+        - Alchemist
+    Mechanic: Create a Potion of Healing. After successfully completing a Quick Rest, you can exchange one Rest Card for a Potion of Healing without expending Power Points. All Potions created last until the end of your next Full Rest.
+    Description: It's always good to have some spare healing on hand. 
+    XP: 2
+    PP: 1
+    Prereq:
+        Role: Support
+        Level: 2
+        Skill: Craft > 0
+
+Create Poison:
+    Type: Major
+    Category:
+        - Combat
+        - Alchemist
+    Mechanic: Create a Poison Bottle. Apply to a Physical Weapon to force a DR 2 VIT Save, on a failure they take 1 damage bypassing AP. All Poison Bottles created last until the end of your next Full Rest.
+    Description: It's not wise to get on someone's bad side, especially when they have friends. 
+    XP: 2
+    PP: 1
+    Prereq:
+        Role: Support
+        Level: 3
+        Skill: Craft > 1
+        Power: Create Potion
+    Duration: 1 minute
+
+Potent Brew: 
+    Type: Passive
+    Category: 
+        - Recovery
+        - Alchemist   
+    Mechanic: When creating a consumable item, increase the effect by 1. 
```

Comment:
Are there any restrictions on who can and can't make consumable items? Does the cost in that table refer to purchasing price or crafting ingredient price?

Reply:
Adjusted language to the following: 

`Mechanic: When using a Power that creates a consumable, increase the damage healed or dealt by 1.`